wormhole-connect/src/config/mainnet.ts

```diff
@@ -103,6 +114,7 @@ export const MAINNET_TOKENS: { [key: string]: TokenConfig } = {
     color: '#62688F',
     decimals: 18,
     solDecimals: 8,
+    suiDecimals: 8,
```

Comment:
I know you brought this up earlier, maybe refactor to:
```
decimals: {
  default: 18,
  solana: 8
  sui: 8
}
```
Or rather... maybe we should add a method that figures this out programmatically based on network and token

Reply:
I like the second option